### DISCUSSION TITLE
Goal line in all charts

### DISCUSSION THREAD

**Discussion by siddizi** [chartbrew/chartbrew#256] [Ideas]
Hi!
I want to set goal line in my chart. For example, i can type number in "Goal" parameters after that I want to see line in my chart(see picture). 
What do you think about it?
![image](https://github.com/user-attachments/assets/f44233a3-6062-41e5-89aa-decf073be0d6)

**Comment by razvanilin** [comment]
Hey @siddizi that's a really good suggestion. I placed it under review on our roadmap: https://chartbrew.productroad.com/board/feature-requests/visualize-goals-on-charts/

**Comment by siddizi** [comment]
Hey @razvanilin . Thanks a lot!
